storybooknative/src/InputField/index.js

```diff
@@ -165,17 +203,24 @@ class InputField extends React.Component<Props, State> {
                 {label}
               </StyledInlineLabel>
             )}
-          <StyledTextInput
-            underlineColorAndroid="transparent"
-            autoCorrect={false}
-            placeholderTextColor={defaultTokens.orbit.colorPlaceholderInput}
-            editable={!disabled}
-            size={size}
-            type={type}
-            {...rest}
-            onFocus={this.onFocus}
-            onBlur={this.onBlur}
-          />
+          <StyledInputWrapper>
+            {!!this.props.placeholder &&
+              this.state.displayPlaceholder && (
+                <StyledPlaceholder size={size}>{placeholder}</StyledPlaceholder>
+              )}
+            <StyledTextInput
+              underlineColorAndroid="transparent"
```

Comment:
What do you think about extracting platform specific code to other files?

Reply:
@tbergq what do you think about this?

Reply:
underlineColorAndroid is a prop on the input from react-native. So I would leave it here. 

Generally, we should be very careful with having platform specific files, since it often leads to duplication of code. I find it useful if we have two files when using different native dependencies, like DatePicker component from mobile repo. In this case, they cannot be in the same file. 

at some point we might want to add the PlatformStyleSheet in our styled components wrapper 🤔

Reply:
Fair enough, I'll accept it as it is then. 

Regarding `PlatformStylesheet`, @FilipMessa is tackling it in his PR https://github.com/kiwicom/orbit-components/pull/550